docs/db/changelog/changesets/powerauth-push-server/1.4.x/20230321-init-db.xml

```diff
@@ -50,7 +50,7 @@
             </not>
         </preConditions>
         <comment>Create a new sequence sequence push_campaign_user_seq</comment>
-        <createSequence sequenceName="push_campaign_user_seq"/>
+        <createSequence sequenceName="push_campaign_user_seq" startValue="1" incrementBy="1" cacheSize="20"/>
```

Comment:
@zcgandcomp So do we have to specify the `incrementBy`? Is it also platform-specific?